opennms-doc/guide-admin/src/asciidoc/text/meta-data.adoc

```diff
@@ -35,59 +51,84 @@ The following keys are available under this context:
+Placeholders start with `${` and end with `}` containing a reference to a context-key pair. 
+You may choose to define multiple fallback context-key pairs and and an optional trailing default value.
+Separate context and key by a `:`.
+Use a `|` to separate optiional fall back context-key pairs and default value.  
 
-Each expressions can contain multiple references to meta-data which will be replaced with the according value during evaluation.
-Placeholders starts with `${` and ends with `}` containing a reference to a context-key-pair optionally allowing to define multiple fallback context-key-pairs and and optional trailing default value.
-Context and key are separated by a `:`.
+*Examples*
 
 `${requisition:username}`::
 Will resolve to the `username` as defined in the requisitioning UI or an empty value, if there is no such username defined.
 
-An placeholder can contain an optional default value which is separated by a `|`.
+A placeholder can contain an optional default value which is separated by a `|`.
 
 `${requisition:username|admin}`::
 Will resolve to the `username` as defined in the requisitioning UI or to the value `admin`, if there is no such username defined.
 
-There can be zero or more fallback context-key-pairs in an placeholder which will follow the primary context-key-pair.
-Each fallback context-key-pair is separated by a `|`.
+Use fallback context-key pairs in a placeholder after the primary context-key pair to specify other values if the primary context-key pair is not defined. 
+Separate each fallback context-key-pair by a `|`.
 
 `${requisition:username|requisition:account|admin}`::
 Will resolve to the `username` as defined in the requisitioning UI.
-If there is no such username defined the fallback `account` will be used.
-If both of them do not exist, the fallback value `admin` will be used.
+If there is no such username defined, the fallback `account` will be used.
+If neither exist, the fallback value `admin` will be used.
 
-To resolve the value associated with context-key-pair, the DSL uses scopes which will determine the resolution order.
-The last scope will be queried first and if a scope does not contain the queried context-key-tuple, the next one will be queried.
-Which scopes are available depends on the environment for which a expression is evaluated and is documented in the according places.
-Some environments also provide additional scopes which are not backed by the persisted meta-data but provide additional meta-data related to the current evaluation.
+To resolve the value associated with context-key pair, the DSL uses scopes that will determine the resolution order.
+The last scope will be queried first and if a scope does not contain the queried context-key tuple, the next one will be queried.
+#QUESTION: How does the user know what the last scope is? is there a default order depending on what scopes are being queried?#
```

Comment:
Currently we have the scopes node, interface and service. If I query a metadata entry of an service entity the resolution is service metadata -> interface.metadata -> node metadata. On interface level just interface metadata -> node metadata. On node level only the node is searched for the metadata entry.